pkg/resources/jobs/helpers.go

```diff
@@ -114,3 +116,37 @@ func newIstioEnvVar(istio v1alpha1.K6Scuttle, istioEnabled bool) []corev1.EnvVar
 	}
 	return env
 }
+
+func getInitContainers(k6InitContainerList []v1alpha1.InitContainer, script *types.Script) []corev1.Container {
+	var initContainers []corev1.Container
+
+	for i, k6InitContainer := range k6InitContainerList {
+		initContainer := corev1.Container{
+			Name:                     fmt.Sprintf("k6-init-%d", i),
+			Image:                    k6InitContainer.Image,
+			Command:                  k6InitContainer.Command,
+			Args:                     k6InitContainer.Args,
+			WorkingDir:               k6InitContainer.WorkingDir,
+			Ports:                    nil,
+			EnvFrom:                  k6InitContainer.EnvFrom,
+			Env:                      k6InitContainer.Env,
+			Resources:                corev1.ResourceRequirements{},
+			VolumeMounts:             script.VolumeMount(),
+			VolumeDevices:            nil,
+			LivenessProbe:            nil,
+			ReadinessProbe:           nil,
+			StartupProbe:             nil,
+			Lifecycle:                nil,
+			TerminationMessagePath:   "",
+			TerminationMessagePolicy: "",
+			ImagePullPolicy:          k6InitContainer.ImagePullPolicy,
+			SecurityContext:          nil,
```

Comment:
We cannot use it since in the case of the k6 definition it corresponds to a PodSecurityContext and not a ContainerSecurityContext. In the case of k6 containers it is not explicitly assigned either. I think we can leave the initContainers with the same approach.

Reply:
Yes, that makes sense. If some other use case will appear, this logic might need to be re-visited, but it should be OK for most cases I imagine :+1: